javascript/src/async/zones.js

```diff
@@ -2,10 +2,21 @@ import resync from '../resync'
 import {
     requestZones, receiveZones, failZones, requestGeometry, receiveGeometry, failGeometry
 } from '../actions/zones'
+import { receiveAvailableSpecies } from '../actions/species'
 import { urlEncode } from '../io'
 import config from 'seedsource/config'
 
 
+const availableZoneSelect = ({ runConfiguration }) => {
+    let { point, method } = runConfiguration
+
+    if (point) {
+        point = {x: point.x, y: point.y}
```

Comment:
If point already has `x` and `y` properties, why even do this?

Reply:
Very good question... it was in zoneSelect function below it so I copied the logic figuring it did something useful (I guess it winnows down the variable to what's needed?). Should I remove it from both?

Reply:
Well, I would look into whether there any use to it first, since I've obviously forgotten! Does `point` have anything other than `x` and `y`. If not, then go ahead and remove from both.